.llmwiki/wiki/llm-wiki-design/neutral-llmwiki-root.md

```
@@ -1,14 +1,22 @@
 aliases: [llmwiki-root, neutral-root, codex-fork-fix, dot-llmwiki]
-last_verified: 2026-05-31
+last_verified: 2026-06-01
 status: active
 volatility: stable
 sources: 3
 ---
 
 # Neutral `.llmwiki/` root
 
+> See-also: [[insight-layer-via-hook]]
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)